src/components/homepage/ChallengeInactiveInfo.vue

```diff
@@ -0,0 +1,70 @@
+<script lang="ts">
+/**
+ * ChallengeInactiveInfo Component
+ *
+ * @description
+ * Renders information about an inactive challenge, informing the user
+ * that the challenge they're trying to access is not currently available.
+ *
+ * @example
+ * <challenge-inactive-info />
+ *
+ * @see [Figma Design](https://www.figma.com/design/L8dVREySVXxh3X12TcFDdR/Do-pr%C3%A1ce-na-kole?node-id=6385-26491&t=Kg5tEX1Skno3i6DW-1)
+ */
+
+// libraries
+import { colors } from 'quasar';
+import { defineComponent } from 'vue';
+
+export default defineComponent({
+  name: 'ChallengeInactiveInfo',
+  setup() {
+    // colors
+    const { getPaletteColor, changeAlpha } = colors;
+    const white = getPaletteColor('white');
+    const whiteOpacity20 = changeAlpha(white, 0.2);
```

Comment:
Use global app var `colorWhiteBackgroundOpacity` https://github.com/auto-mat/ride-to-work-by-bike-frontend/blob/d1de13093894012841bd1b4c51e486d97e093a4a/ride_to_work_by_bike_config.toml#L14 here instead of custom defined opacity number, please.